### ISSUE TITLE
cannot reproduce

### ISSUE THREAD

**Issue by RRooyyCChheenn** [bartn8/depthanyevent#4]
Hello, I have a small question and I’m not sure if the authors encountered this issue during their experiments.

Since the authors did not share the training pipeline, I tried to reproduce the distillation framework strictly following the description in the paper. However, I found that using the provided loss functions and models, whether DAV2 or REC_DEV2, I was unable to achieve similar performance. The network seems to converge to a degenerate solution, producing depth maps that are mostly uniform in color and lacking any meaningful depth structure.

I was wondering if the authors have any insights or suggestions on this issue.

**Comment by bartn8** [comment]
Hi RRooyyCChheenn,

Sorry for the late response. I suggest you to check the input and output of the model: in particular, input tencode, and proxy labels. I suggest you to add some debug images that plot both input and output. Check also the loss: https://github.com/uzh-rpg/rpg_e2depth/blob/master/e2depth/model/loss.py is a good start.

Best,
Luca
